subgraphs/isle-finance-v0/protocols/isle-finance-v0/src/constants.ts

```diff
@@ -20,26 +20,25 @@ export const ETH_ADDRESS = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee";
 export const ETH_NAME = "Ether";
 export const ETH_SYMBOL = "ETH";
 export const ETH_DECIMALS = 18;
-export const DEFAULT_DECIMALS = 18;
+export const ISLE_USD_DECIMALS = 6;
```

Comment:
I think the default decimals of an ERC20 contract from OZ is 18. We'll have to change to 6 for USDC in production.
However, I think we might just stay with 18 for simplicity.